Gemfile

```diff
@@ -59,7 +59,7 @@ gem 'concurrent-ruby'
 gem 'vault', git: 'https://github.com/zendesk/vault-ruby.git', ref: '96be391a2fd50a42871c8b9dc3c59fddbdbdc556'
 gem 'lograge'
 gem 'logstash-event'
-gem 'hashdiff', git: 'https://github.com/liufengyun/hashdiff.git' # fails to install on ruby 3.2 from gem when deploying
+gem 'hashdiff'
```

Comment:
There have been a couple of patch releases of `hashdiff` since  the Ruby 3.2 upgrade. I'm expected it to be fixed now.

Let's use the released gem on Rubygems.

Reply:
yeah, nice find :)